rs/nns/integration_tests/src/update_canister_settings.rs

```diff
@@ -36,7 +36,8 @@ fn test_update_canister_settings_proposal(
     let target_memory_allocation = 1u64 << 33;
     let target_compute_allocation = 10u64;
     let target_freezing_threshold = 100_000u64;
-    let target_wasm_memory_limit = 1u64 << 32;
+    let target_wasm_memory_limit = 1u64 << 36;
```

Comment:
Why was this changed? 

```suggestion
    let target_wasm_memory_limit = 1u64 << 32;
```

Reply:
target_wasm_memory_limit needs to be smaller than `target_wasm_memory_threshold`. I also wanted to test whether `target_wasm_memory_threshold` will work with values greater than `u32`, so I needed this.

Reply:
This justification should IMHO be written in a comment.

Reply:
Done.